network-api/networkapi/buyersguide/templates/fragments/product_criterion_primary_info.html

```diff
@@ -6,11 +6,19 @@
   <div class="rating">
     <span><strong>
     {% if value == "U" %}
+      {% if show_value_as_symbol == "True" %}
```

Comment:
let's make this a true boolean, so we can just say `{% if show_value_as_symbol %}`